src/dfcx_scrapi/core/conversation.py

```diff
@@ -493,13 +561,17 @@ def reply(
             logging.error("query_params %s", query_params)
             logging.error("query_input %s", query_input)
             logging.error(traceback.print_exc())
-            retries += 1
-            if retries < MAX_RETRIES:
-                logging.error("retrying")
-                return self.reply(send_obj, restart=restart, retries=retries)
-            else:
-                logging.error("MAX_RETRIES exceeded")
-                return {}
+            return {
```

Comment:
Add error handling returns LGTM per our discussion.   
However, let's put the MAX_RETRIES handling back in there for now until we can do a larger refactor

Reply:
Added back MAX_RETRIES handling